src/webui/src/components/Modals/ChangeColumnComponent.tsx

```diff
@@ -41,7 +47,7 @@ class ChangeColumnComponent extends React.Component<ChangeColumnProps, ChangeCol
         } else {
             if (source.includes(label)) {
                 // remove from source
-                const result = source.filter((item) => item !== label);
+                const result = source.filter(item => item !== label);
```

Comment:
I think this is less clear to human. It's hard to figure out associative order at first glance to me.

Reply:
alwaysParens: `avoid | always`;
default val is avoid, not add () for arrow function.
So we change it to `always`?